pkg/GAPJulia/JuliaInterface/julia/gap.jl

```diff
@@ -68,3 +68,52 @@ for (left, right) in typecombinations
         end
     end
 end
+
+macro gap(str)
+    return EvalString(string(str))
+end
+
+export @gap
+
+function LoadPackageAndExposeGlobals(package::String, mod::String; all_globals::Bool = false)
+    mod_sym = Symbol(mod)
+    Base.MainInclude.eval(:(
+        module $(mod_sym)
+            import GAP
+        end
+    ))
+    mod_mod = Base.MainInclude.eval(:(Main.$(mod_sym)))
+    Base.invokelatest(LoadPackageAndExposeGlobals,package,mod_mod; all_globals=all_globals)
+end
+
+function LoadPackageAndExposeGlobals(package::String, mod::Module = Main; all_globals::Bool = false )
+    if mod == Main
+        eval_func = Base.MainInclude.eval
+    else
+        eval_func = mod.eval
+    end
+    current_gvar_list = EvalString("ShallowCopy(NamesGVars())")
+    load_package = EvalString("LoadPackage(\"$package\")")
+    if load_package == Globals.fail
+        error("cannot load package $package")
+    end
+    new_gvar_list = nothing
+    if all_globals
+        new_gvar_list = Globals.NamesGVars()
+    else
+        new_gvar_list = Globals.Difference(Globals.NamesGVars(),current_gvar_list)
+    end
+    new_gvar_list_translated = gap_to_julia(Array{String,1},new_gvar_list)
```

Comment:
Why not directly like this:
```suggestion
    new_symbols = gap_to_julia(Array{Symbol,1},new_gvar_list)
```